src/ui/utils/prefs.ts

```diff
@@ -23,7 +23,7 @@ pref("devtools.hitCounts", "hide-counts");
 pref("devtools.features.columnBreakpoints", false);
 pref("devtools.features.commentAttachments", false);
 pref("devtools.features.enableLargeText", false);
-pref("devtools.features.enableNewComponentArchitecture", false);
+pref("devtools.features.disableNewComponentArchitecture", false);
```

Comment:
Initializing _disable_ to `false` means all users will get the new console by default.